src/components/toolCard/index.tsx

```diff
@@ -9,11 +9,11 @@ import {
     TagLeftIcon,
     TagLabel,
     Flex,
-    Button,
     IconButton,
     Tooltip,
 } from "@chakra-ui/react";
 import { CheckIcon, DeleteIcon, StarIcon } from "@chakra-ui/icons";
+import { Button } from "..";
```

Comment:
let's remove this unused `useState` on line 1 (sorry can't comment on that directly :?)